### DISCUSSION TITLE
[@types/node] error TS2709: Cannot use namespace '_URLSearchParams' as a type.

### DISCUSSION THREAD

**Discussion by elsuxo** [DefinitelyTyped/DefinitelyTyped#58603] [Issues with a @types package]
after `npm update` our angular project, **@types/node** is updated to version **12.20.43** and throws these errors when compiling:

```
Error: node_modules/@types/node/url.d.ts:120:43 - error TS2709: Cannot use namespace '_URLSearchParams' as a type.

120         interface URLSearchParams extends _URLSearchParams {}
                                              ~~~~~~~~~~~~~~~~


Error: node_modules/@types/node/url.d.ts:121:31 - error TS2709: Cannot use namespace '_URL' as a type.

121         interface URL extends _URL {}
                                  ~~~~
```

i tried updating to latest version 17.0.14 but i get the same errors. 
Is this a known issue? i did not find anything regarding these error messages.

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment]
I believe you're seeing outcome of this change:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/58277/files
what's the @types/node usage within your project? (I believe it's not there by default)

**Reply by elsuxo** [reply]
its an angular project.. the project was created with `ng new` which also created the dev dependency for the @types/node package

**Comment by peterblazejewicz** [comment]
OK, I understand, but rather would be nice to get some information, for example it was found that using 'lib.webworker' as lib dependency causes recent changes in types/node to throw that TSC error
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/58619
